### ISSUE TITLE
Hide <Pagination> UI when only a single page exists

### ISSUE THREAD

**Issue by jscottsmith** [gumgum/gumdrops#42]
Pagination UI should not be displayed if there's only a single page, avoiding this:

```jsx
{advertisers.totalPages > 1 && (
    <Pagination
        activePage={activePage}
        lastPage={advertisers.totalPages}
        boundaries
        justify
        onChange={this._handlePageChange}
        size="sm"
    />
)}
```

**Comment by asifsha** [comment]
Hello,
I can do this, where I need to implement it ?

**Comment by piyush-bhati** [comment]
I would like to work on it

**Comment by asifsha** [comment]
Fixed in https://github.com/gumgum/gumdrops/issues/51

**Comment by jscottsmith** [comment]
Published in [1.6.1](https://github.com/gumgum/gumdrops/releases/tag/v1.6.1)
